bpfctl/src/main.rs

```diff
@@ -211,24 +232,58 @@ impl ProgTable {
         let mut table = Table::new();
 
         table.load_preset(comfy_table::presets::NOTHING);
-        table.set_header(vec!["UUID", "Type", "Name", "Location", "Metadata"]);
+        table.set_header(vec!["Kernel_ID", "Bpfd_UUID", "Name", "Type", "Load_Time"]);
         ProgTable(table)
     }
 
     fn add_row(
         &mut self,
+        kernel_id: String,
         uuid: String,
-        type_: String,
         name: String,
-        location: String,
-        metadata: String,
+        type_: String,
+        load_time: String,
     ) {
-        self.0.add_row(vec![uuid, type_, name, location, metadata]);
+        self.0
+            .add_row(vec![kernel_id, uuid, name, type_, load_time]);
     }
 
     fn add_response_prog(&mut self, r: list_response::ListResult) -> anyhow::Result<()> {
+        self.add_row(
+            r.bpf_id.to_string(),
+            r.id.unwrap_or("".to_string()),
+            r.name,
+            (ProgramType::try_from(r.program_type)?).to_string(),
+            r.loaded_at,
+        );
+
+        Ok(())
+    }
+}
+
+impl std::fmt::Display for ProgTable {
+    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+        write!(f, "{}", self.0)
+    }
+}
+
+fn print_get(r: &list_response::ListResult) -> anyhow::Result<()> {
+    // if program is managed by bpfd print UUID, Location and Metadata
+    let bpfd_info = if let Some(uuid) = r.clone().id {
         let prog_type: ProgramType = r.program_type.try_into()?;
-        match prog_type {
+        let location = match &r.clone().location.unwrap() {
+            // Cast imagePullPolicy into it's concrete type so we can easily print.
+            Location::Image(i) => format!(
+                r#"Image:                              {} 
+ImagePullpolicy:                    {}"#,
+                i.url,
+                TryInto::<ImagePullPolicy>::try_into(i.image_pull_policy)?
+            ),
+            Location::File(p) => format!(r#"Path:                           {p}"#),
```

Comment:
```suggestion
            Location::File(p) => format!(r#"Path:                               {p}"#),
```
Looks like for a file, you need a few more spaces:
```
$ bpfctl get 5862

#################### Bpfd State ####################

UUID:                               6ea3e8b2-5bfe-45f3-8968-da1aefd9f106                              
Path:                           /run/bpfd/examples/go-xdp-counter/bpf_bpfel.o             
Priority:                           100
Iface:                              vethff657c7
:
```

Reply:
Ah great catch